apps/calendar/js/views/single_month.js

```diff
@@ -8,7 +8,7 @@ var daysBetween = Calc.daysBetween;
 var daysInWeek = Calc.daysInWeek;
 var getDayId = Calc.getDayId;
 var spanOfMonth = Calc.spanOfMonth;
-var startsOnMonday = Calc.startsOnMonday;
+var startDay = Calc.startDay;
```

Comment:
don't cache `startDay` here since it can change during `localized` event and we rebuild the month when that happens.